### PR TITLE
contact us section

### PR DESCRIPTION
**Pull request by Anand-Madhu123**
contact 6

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -485,57 +485,64 @@ <h5 class="pa-h1-v3 pa-bright pt-4 pb-1 mb-4 fadeInDelay03Duration10">Let <stron
...
-                        </div>
+                        <h1 class="pa-h1-v3 pa-bright fadeInDelay03Duration10"><strong>Our Address</strong></h1><br>
+                        <p class="pa-p-v1 pa-white fadeInDelay03Duration10">BrainChild Services</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay06Duration10">Cochin,Kerala</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay10Duration10"><span><i class="material-icons">mail</i></span>contact@brainchildservices.com</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay10Duration10"><span><i class="material-icons">location_on</i></span>Google Map</p>
+                        <div>
+                            
+                                <iframe width="400" height="450"
+                                    src="https://maps.google.com/maps?q=chottanikkara,Kerala&t=k&z=11&ie=UTF8&iwloc=&output=embed"
```
Comment (VishnuKulakkattu) [inline index.html]: Wrong Location

File: index.html [hunk 2]
```diff
@@ -485,57 +485,64 @@ <h5 class="pa-h1-v3 pa-bright pt-4 pb-1 mb-4 fadeInDelay03Duration10">Let <stron
...
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-behance"></i></span></a>
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-dribbble"></i></span></a>
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-instagram"></i></span></a>
-                            <p class="pa-p-v1 pa-white pa-contact-lastline fadeInDelay03Duration10">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-                        </div>
+                        <h1 class="pa-h1-v3 pa-bright fadeInDelay03Duration10"><strong>Our Address</strong></h1><br>
+                        <p class="pa-p-v1 pa-white fadeInDelay03Duration10">BrainChild Services</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay06Duration10">Cochin,Kerala</p>
```
Comment (wandercode) [inline index.html]: add a better icon.

File: index.html [hunk 3]
```diff
@@ -485,57 +485,64 @@ <h5 class="pa-h1-v3 pa-bright pt-4 pb-1 mb-4 fadeInDelay03Duration10">Let <stron
...
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-behance"></i></span></a>
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-dribbble"></i></span></a>
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-instagram"></i></span></a>
-                            <p class="pa-p-v1 pa-white pa-contact-lastline fadeInDelay03Duration10">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-                        </div>
+                        <h1 class="pa-h1-v3 pa-bright fadeInDelay03Duration10"><strong>Our Address</strong></h1><br>
+                        <p class="pa-p-v1 pa-white fadeInDelay03Duration10">BrainChild Services</p>
```
Comment (wandercode) [inline index.html]: add a better icon

File: index.html [hunk 4]
```diff
@@ -485,57 +485,64 @@ <h5 class="pa-h1-v3 pa-bright pt-4 pb-1 mb-4 fadeInDelay03Duration10">Let <stron
...
+                        <p class="pa-p-v1 pa-white fadeInDelay06Duration10">Cochin,Kerala</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay10Duration10"><span><i class="material-icons">mail</i></span>contact@brainchildservices.com</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay10Duration10"><span><i class="material-icons">location_on</i></span>Google Map</p>
```
Comment (wandercode) [inline index.html]: instead of google map ----> "Reach Us"

File: index.html [hunk 5]
```diff
@@ -485,57 +485,64 @@ <h5 class="pa-h1-v3 pa-bright pt-4 pb-1 mb-4 fadeInDelay03Duration10">Let <stron
...
+                                    </div>
                     </div>
-                </div>
+                    </div>
+                
                 <!-- Contact Form -->
                 <div class="col-12 col-md-6 col-xl-5 pa-contact-form d-flex justify-content-end">
                     <div class="pa-glass pa-right">
-                        <h1 class="pa-h1-v3 pa-bright fadeInDelay03Duration10">Send us a <strong>Message</strong></h1>
+                        <h1 class="pa-h1-v3 pa-bright fadeInDelay03Duration10"><strong>Add your Contact details here and send us a message </strong></h1>
```
Comment (wandercode) [inline index.html:516]: after this content give a proper space from the form.
image

**Review by wandercode — CHANGES_REQUESTED** [review]

File: index.html [hunk 6]
```diff
@@ -485,57 +485,64 @@ <h5 class="pa-h1-v3 pa-bright pt-4 pb-1 mb-4 fadeInDelay03Duration10">Let <stron
             <!-- Section Header -->
             <div class="row text-center">
                 <div class="col-12 col-xl-10 offset-xl-1">
-                    <h1 class="pa-h1-v2 pa-bright fadeInDelay00Duration10">Let's <strong>Connect</strong></h1>
-                    <p class="pa-p-v1 pa-white fadeInDelay03Duration10">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec odio. Praesent libero. Sed cursus ante dapibus diam. Sed nisi. Nulla quis sem at nibh elementum imperdiet. Duis sagittis ipsum. Praesent mauris.</p>
+                    <h1 class="pa-h1-v2 pa-bright fadeInDelay00Duration10"><strong>Contact Us</strong></h1>
```
Comment (wandercode) [inline index.html]: please make it 
image

**Review by wandercode — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (Anand-Madhu123) [reply index.html]: done

File: index.html [hunk 7]
```diff
@@ -486,56 +486,63 @@ <h5 class="pa-h1-v3 pa-bright pt-4 pb-1 mb-4 fadeInDelay03Duration10">Let <stron
...
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-instagram"></i></span></a>
-                            <p class="pa-p-v1 pa-white pa-contact-lastline fadeInDelay03Duration10">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-                        </div>
+                        <h1 class="pa-h1-v3 pa-bright fadeInDelay03Duration10"><strong>Our Address</strong></h1><br>
+                        <p class="pa-p-v1 pa-white fadeInDelay06Duration10"><span><i class="material-icons">location_on</i></span>Brainchildservices</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay06Duration10"><span><i class="material-icons">location_on</i></span>Cochin,Kerala</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay10Duration10"><span><i class="material-icons">mail</i></span>contact@brainchildservices.com</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay10Duration10"><span><i class="material-icons">location_on</i></span>Reach Us</p>
```
Comment (wandercode) [inline index.html]: please give proper icons here like below
image

**Review by wandercode — CHANGES_REQUESTED** [review]
add mobile number also

File: index.html (hunk 2 above)
Comment (Anand-Madhu123) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (Anand-Madhu123) [reply index.html]: done

File: index.html (hunk 5 above)
Comment (Anand-Madhu123) [reply index.html:516]: done

File: index.html (hunk 6 above)
Comment (Anand-Madhu123) [reply index.html]: done

File: index.html (hunk 7 above)
Comment (Anand-Madhu123) [reply index.html]: done

File: index.html [hunk 8]
```diff
@@ -486,56 +486,63 @@ <h5 class="pa-h1-v3 pa-bright pt-4 pb-1 mb-4 fadeInDelay03Duration10">Let <stron
...
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-dribbble"></i></span></a>
-                            <a class="pa-social-icons" href="https://www.facebook.com/SkilltechWebDesign" target="_blank"><span class="fadeInDelay03Duration10"><i class="fa fa-instagram"></i></span></a>
-                            <p class="pa-p-v1 pa-white pa-contact-lastline fadeInDelay03Duration10">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-                        </div>
+                        <h1 class="pa-h1-v3 pa-bright fadeInDelay03Duration10"><strong>Our Address</strong></h1><br>
+                        <p class="pa-p-v1 pa-white fadeInDelay06Duration10"><span><i class="fa fa-building"></i></span>Brainchildservices</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay06Duration10"><span><i class="fa fa-address-book"></i></span>Cochin,Kerala</p>
+                        <p class="pa-p-v1 pa-white fadeInDelay10Duration10"><span><i class="fa fa-envelope"></i></span>contact@brainchildservices.com</p>
```
Comment (wandercode) [inline index.html]: please add our mobile number here before mail id

**Review by wandercode — CHANGES_REQUESTED** [review]

File: index.html (hunk 8 above)
Comment (Anand-Madhu123) [reply index.html]: done

**Review by wandercode — APPROVED** [review]

**Review by VishnuKulakkattu — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Anand-Madhu123) [reply index.html]: done

**Pull request merged**